databases/pgcli/Portfile

```diff
@@ -21,15 +21,12 @@ checksums           rmd160  0df7f981395f69e47b277d52336b117ecead12a4 \
                     sha256  65669f869766bb3416dc26ea719c65bd3297ba692207aa9e0f2164d26315ca8d \
                     size    444871
 
-variant python27 conflicts python36 python37 python38 python39 description "Use Python 2.7" {}
```

Comment:
the commit message here should probably mention "variant" here instead of "subport"?